File: backend/src/routes/health.js
'use strict'

const express = require('express')
const config  = require('../config')

const router = express.Router()

function healthHandler(req, res) {
  res.json({
    status:    'ok',
    service:   'serverpilot-backend',
    version:   '0.2.0',
    env:       config.appEnv,
    timestamp: new Date().toISOString(),
  })
}

router.get('/health',     healthHandler)
router.get('/api/health', healthHandler)

module.exports = router

File: backend/internal/handler/health.go
package handler

import (
	"net/http"
	"time"

	"github.com/serverpilot/backend/internal/config"
)

func Health(cfg config.Config) http.HandlerFunc {
	return func(w http.ResponseWriter, r *http.Request) {
		writeJSON(w, http.StatusOK, map[string]any{
			"status":    "ok",
			"service":   "serverpilot-backend",
			"version":   "0.1.0",
			"env":       cfg.AppEnv,
			"timestamp": time.Now().UTC(),
		})
	}
}
